### DISCUSSION TITLE
What's difference between default.some-hash.css and entry.some-hash.css and index.some-hash.css?

### DISCUSSION THREAD

**Discussion by AaronConlon** [nuxt/nuxt#16411] [Questions] [answered]
Hi.
I am a rookie,I don't know about what's difference between default.some-hash.css and entry.some-hash.css and index.some-hash.css.
Someone can help?Thanks.

**Comment by TheAlexLichter** [comment, marked as answer]
entry -> CSS that is needed when the app starts (usually global styles)
default -> I'd guess the default layout CSS
index -> pages/index.vue CSS

**Reply by AaronConlon** [reply]
Thank you very much.
